cdap-docs/application-templates/source/etl/templates/sources/batch/database.rst

```diff
@@ -23,3 +34,54 @@ specified table. Examples::
 
 *Note:* Please include the same WHERE clauses in this query as the ones used in the import
 query to reflect an accurate number of records to import.
+
+**connectionString:** JDBC connection string including database name.
+
+**tableName:** Table name to read from.
+
+**user:** Use to connect to the specified database as. Required for databases that need
+authentication. Optional for databases that do not require authentication.
+
+**password:** Password to use to connect to the specified database. Required for databases
+that need authentication. Optional for databases that do not require authentication.
+
+**jdbcPluginName:** Name of the JDBC plugin to use. This is the value of the 'name' key
+defined in the JSON file for the JDBC plugin.
+
+**jdbcPluginType:** Type of the JDBC plugin to use. This is the value of the 'type' key
+defined in the JSON file for the JDBC plugin. Defaults to 'jdbc'.
+
+.. rubric:: Example
+
+::
+
+  {
+    "name": "Database",
+    "properties": {
+      "importQuery": "select id,name,email,phone from users",
+      "countQuery": "select count(*) from users",
+      "connectionString": "jdbc:postgresql://localhost:5432/prod",
+      "tableName": "users",
```

Comment:
turns out table name is unused (https://issues.cask.co/browse/CDAP-2735).  I'll remove it from docs and update code in a separate PR